src/main/java/org/openrewrite/analysis/dataflow/DataFlowNode.java

```diff
@@ -65,7 +65,7 @@ public static Option<DataFlowNode> of(Cursor cursor) {
                     .toOption()
                     .map(expr -> new ExpressionDataFlowNode(cursor, expr));
         } else if (cursor.getValue() instanceof J.VariableDeclarations.NamedVariable) {
-            return Parameter.viewOf(cursor).map(parameter -> (DataFlowNode) new ParameterDataFlowNode(cursor, parameter)).toOption();
+            return Parameter.viewOf(cursor).map(DataFlowNode.class::cast).toOption();
```

Comment:
Well this is a bug. :/

Reply:
Logged separately in https://github.com/openrewrite/rewrite-static-analysis/issues/150